OpenSim/Common/APDMDataReader.cpp

```diff
@@ -67,21 +67,25 @@ APDMDataReader::extendRead(const std::string& fileName) const {
     bool newFormat = false;
     if (tokens[0] == "Format=7") {
         newFormat = true;
-        std::vector<std::string> labels;
+        dataRate = 128; // Will fix on reading first 2 rows
         // Header Line 1:Format=7, [I1,,,$IMU1,,,,,,,,,,,]*
         // Header Line 2: Time,[Accelerometer,,,Gyroscope,,,Magnetometer,,,Barometer,Orientation,,,]*
         // Header Line 3: ,[X,Y,Z,X,Y,Z,X,Y,Z,,S,X,Y,Z]*
+        std::vector<std::string> mtLabels;
```

Comment:
i don't understand the reason for the name change. What does "mt" signify?

Reply:
mtLabels stands for EmptyLabels. Old format had suffixes for various quanities e.g. Lumbar/Acceleration, new one has Lumber on one line and Acceleration on a different line, using an array with one entry that is a n empty string allows us to reuse the function to locate the column number corresponding to a specific sensor. If it makes it more clear I can rename it to emptyLabels.